src/vma/dev/net_device_table_mgr.cpp

```diff
@@ -323,10 +333,74 @@ void net_device_table_mgr::verify_ipoib_mode(struct ifaddrs* ifa)
 		vlog_printf(VLOG_WARNING,"This option in no longer needed in this version\n");
 		vlog_printf(VLOG_WARNING,"Please refer to Release Notes for more information\n");
 		vlog_printf(VLOG_WARNING,"************************************************************************\n");
+		return 0;
 	}
 	else {
 		ndtm_logdbg("verified interface '%s' is running with umcast disabled", ifa->ifa_name);
 	}
+	return 1;
+}
+
+bool net_device_table_mgr::check_ib_device_supported(const char* ifname) {
+	if (check_device_exist(ifname, CX4_DRIVER_PATH)) {
```

Comment:
If you only checked 
```
check_qp_creation(ifname, IBV_QPT_UD);
```
(without distinguishing between CX3/CX4 types) could it cover all cards types?

Better remove checking of hard coded path CX4_DRIVER_PATH since it will break when:
1. Future OFED releases will modify this path
2. testing for next generation cards: CX5, CX6,...